docs/benchmarks/data-comparisson.md

```diff
@@ -49,7 +49,7 @@ ETC. AUTOMATIC DOCUMENT CLASSIFICATION, CLUSTERS, CLUSTERING, TERM
 CLASSIFICATIONS. #
 ```
 
-**Relevance assessments:**<br />In `lisarj.num` are the relevance assessments represented with IDs. Every query-ID (first column) is followed by document-IDs that are considered as relevant. Depending on the program used to open or parse the file the relevant document-IDs could continue in the next line. Since there is no end marker it would be best to parse the lines depending on the query-ID.
+**Relevance assessments:**<br />In `lisarj.num` are the relevance assessments represented with IDs. Every query-ID (first column) is followed by the number of relevant documents-IDs before the relevant document-IDs start. Depending on the program used to open or parse the file the relevant document-IDs could continue in the next line. Since there is no end marker it would be best to parse the lines depending on the query-ID and number of relevant documents IDs.
```

Comment:
This is just a test comment to start a review, i think the changes look fine

Reply:
This way I can see the review. I think when I change something and commit it should show here and you can resolve it when it's done  :thumbsup: